1-exercises/A-reading-list/script.js

```diff
@@ -1,7 +1,28 @@
 function readingList(books) {
-  // Write your code here...
+const content = document.querySelector("#content"); // select div tag by id 
+const ulElement = document.createElement("ul"); // create a new ul tag
+for(let i=0 ; i < books.length;i++){
+  // loop  in books array
+  const liElement = document.createElement("li"); //// create a new li tag
+  const images = document.createElement("img"); //// create a new img tag
+  const header = document.createElement("h1"); // // create a new  h1 tag
+
+  header.innerHTML = books[i].title; // display header from titile of object
+  images.src = books[i].bookCoverImage; // git the ing src from bookCoverImage in object
+  images.alt = books[i].author; // git alt to img tag from  bookCoverImage in object
+  if (books[i].alreadyRead == true) { // if the book it's read 
```

Comment:
Excellent job buddy. However this can be improved, instead of using this `if (books[i].alreadyRead == true)` you can just use this and still works the same but more cleaver way `if (books[i].alreadyRead)` you don't need to say `==true` the if condition will figure it out by itself!!

Reply:
Thanks Altom for reviewing my code , yes this is great idea i will doing that in this week 
Thanks